### PR TITLE
week-1 submissions

### PR DESCRIPTION
**Pull request by samarthkathal**

### CODE REVIEW

**Comment by NishantNaithani** [comment]
@samarthkathal ,

Full assignment is not updated please have a look.

thanks

**Comment by samarthkathal** [comment]
I have updated the PR @NishantNaithani 
Please have a look, thanks.

File: Week-1/assignmant-2/hometown/index.html
Comment (NishantNaithani) [inline Week-1/assignmant-2/hometown/index.html:1]: Here always wrap your webpage content under a parent tag,
For e.g. `<main><p>......</p></main>`

File: Week-1/assignmant-2/html-tables/index.html
Comment (NishantNaithani) [inline Week-1/assignmant-2/html-tables/index.html:1]: Here `<p></p>` tag is meant for other types of data but here you wrapped `<table></table>` under the `<p></p>` tag, which violates the semantic HTML concept.

File: Week-1/assignmant-2/news-articles/index.html
Comment (NishantNaithani) [inline Week-1/assignmant-2/news-articles/index.html:1]: Here `required` attribute for `Name` and `Email` is not used.

File: Week-1/assignmant-2/poem/index.html
Comment (NishantNaithani) [inline Week-1/assignmant-2/poem/index.html:1]: Here always wrap your webpage content under a parent tag,
For e.g. `<main><p>......</p></main>`

**Comment by NishantNaithani** [comment]
@samarthkathal 
Added comments, please have a look.

**Review by NishantNaithani — CHANGES_REQUESTED** [review]

**Comment by samarthkathal** [comment]
Thanks @NishantNaithani , I've Updated the branch with the suggested changes. Please have a look

File: Week-1/assignmant-2/html-tables/index.html
Comment (NishantNaithani) [reply Week-1/assignmant-2/html-tables/index.html:1]: Here always wrap your webpage content under a parent tag,
For e.g. `<main><p>......</p></main>`

**Review by NishantNaithani — APPROVED** [review]

**Comment by NishantNaithani** [comment]
@samarthkathal ,
Added a small comment

Thanks

File: Week-1/assignmant-2/html-tables/index.html
Comment (NishantNaithani) [inline Week-1/assignmant-2/html-tables/index.html:1]: Here always wrap your webpage content under a parent tag,
For e.g. `<main><p>......</p></main>`

**Review by NishantNaithani — CHANGES_REQUESTED** [review]

File: Week-1/assignmant-2/html-tables/index.html
Comment (samarthkathal) [reply Week-1/assignmant-2/html-tables/index.html:1]: fixed it now, thanks

**Comment by samarthkathal** [comment]
@NishantNaithani I've added the `<main>` tag now, please check
thanks

**Review by NishantNaithani — APPROVED** [review]

**Review by NishantNaithani — APPROVED** [review]

**Pull request merged**